wls-gui-wahllokalsystem/src/components/vorfaelleundvorkommnisse/TheEreignisseRow.vue

```diff
@@ -10,7 +10,12 @@
         >{{ index + 1 }}</v-col
       >
       <v-col cols="2">
-        <base-time-input v-model="ereignis.uhrzeit"></base-time-input>
+        <base-time-input
+          :model-value="ereignis.uhrzeit"
+          @update:model-value="
+            (value) => onEreignisUhrzeitUpdateModelValue(value, index)
+          "
+        ></base-time-input>
```

Comment:
```suggestion
        />
```

das kommt auch an anderen stellen noch vor die ich nicht kommentieren kann. bitte nochmal prüfen wo man das noch anpassen kann. macht den code lesbarer finde ich, weil wir uns doppelte informationen sparen